includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -491,11 +497,13 @@ public function process_redirect_payment( $order_id, $intent_id, $save_payment_m
 	 * @return array
 	 */
 	public function get_payment_fields_js_config() {
-		$payment_fields                         = parent::get_payment_fields_js_config();
-		$payment_fields['accountDescriptor']    = $this->get_account_statement_descriptor();
-		$payment_fields['addPaymentReturnURL']  = wc_get_account_endpoint_url( 'payment-methods' );
-		$payment_fields['gatewayId']            = self::GATEWAY_ID;
-		$payment_fields['paymentMethodsConfig'] = $this->get_enabled_payment_method_config();
+		$payment_fields                           = parent::get_payment_fields_js_config();
+		$payment_fields['accountDescriptor']      = $this->get_account_statement_descriptor();
+		$payment_fields['addPaymentReturnURL']    = wc_get_account_endpoint_url( 'payment-methods' );
+		$payment_fields['gatewayId']              = self::GATEWAY_ID;
+		$payment_fields['paymentMethodsConfig']   = $this->get_enabled_payment_method_config();
+		$payment_fields['upeAppareance']          = get_transient( self::UPE_APPEARANCE_TRANSIENT );
```

Comment:
```suggestion
		$payment_fields['upeAppearance']          = get_transient( self::UPE_APPEARANCE_TRANSIENT );
```
Apologising again about this one, but you know I had to do it.